packages/dev/docs/src/docs.css

```diff
@@ -30,6 +30,46 @@ html, body {
   }
 }
 
+.heroImage {
+  display: flex;
+  justify-content: center;
+  padding: 40px 0;
+
+  img {
+    max-width: 100%;
+  }
+}
+
+.highlights {
+  display: flex;
+  flex-wrap: wrap;
+
+  @media (max-width: 1020px) {
+    display: block;
+
+    div {
+      width: 100%;
+    }
+  }
+
+  div {
+    box-sizing: border-box;
+    padding: 0 16px;
+  }
+
+  @media (min-width: 1020px) {
+    div:first-child:nth-last-child(3),
+    div:first-child:nth-last-child(3) ~ div {
+      width: 33.3333%;
```

Comment:
👀 
wha? why?

Reply:
magic! if 3 items then width is 33%, otherwise width is different (see later code)